runtime/src/extensions.rs

```diff
@@ -99,62 +95,86 @@ impl TryFrom<u16> for v0::FuncId {
 	}
 }
 
+mod utils {
```

Comment:
Subjective, but dont see the benefit to a module here. As a maintainer adding a new extension function, I need to now know that I need to call such function within some utils submodule, adding to the complexity rather than just a function that exists within the extension module itself. Utils seems more broad, in general usage beyond this extension which isnt the case.

I can see more benefit in some `dispatch` module now that the dispatch functionality is becoming overloaded. Likewise some module specific to reading state as those use cases grow.

Reply:
I removed the utils module. I request we wait on moving dispatch out to a separate module until we start the Pop API pallet. The reason being that the extension should be come simpler as we move functionality to the pallet.